modin/pandas/io.py

```diff
@@ -130,6 +154,94 @@ def read_csv(
+    |                         |                 |                | Non-lazy       | Parameter is unsupported only    |
+    |                         |                 |                |                | if `skiprows` and `header`       |
+    |                         |                 |                |                | values have intersections        |
+    |                         |                 |                |                | **OmniSci**:                     |
+    |                         |                 |                |                | Only integer values are          |
+    |                         |                 |                |                | supported                        |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | skip_blank_lines        |                 |                | Partial        | **OmniSci**:                     |
+    |                         |                 |                | Non-lazy       | Only True value is supported     |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | parse_dates             |                 |                | Partial        | **OmniSci**:                     |
+    |                         |                 |                | Non-lazy       | Bool and flattened list of       |
+    |                         |                 |                |                | string column names are          |
+    |                         |                 |                |                | supported                        |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | chunksize               | Harmful         | Harmful        | Harmful        |                                  |
+    |                         |                 |                | Non-lazy       |                                  |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | compression             |                 |                | Non-lazy       | **OmniSci**:                     |
+    |                         |                 |                |                | `compression` is inferred        |
+    |                         |                 |                |                | automatically and shouldn't be   |
+    |                         |                 |                |                | specified                        |
+    +-------------------------+-----------------+----------------+----------------+----------------------------------+
+    | index_col               |                 |                | Harmful        | These parameters are not         |
+    | squeeze                 |                 |                | Non-lazy       | supported by OmniSci execution   |
```

Comment:
Not sure we should put `sep` and `delimiter` parameters there since we forward them to the `pyarrow.csv.read_csv` [here](https://github.com/modin-project/modin/blob/master/modin/experimental/core/execution/native/implementations/omnisci_on_native/io/io.py#L261) (so we support these parameters for OmniSci execution).